base_location_nuts/wizard/nuts_import_view.xml

```diff
@@ -36,7 +35,7 @@
 <menuitem action="nuts_import_action"
           id="nuts_import_menu"
           name="Import NUTS 2013"
-          parent="base.menu_localisation"
+          parent="sales_team.menu_localisation"
```

Comment:
Well, on base_location there is another menu defined: https://github.com/OCA/partner-contact/blob/10.0/base_location/views/better_zip_view.xml#L61